pkg/node/sendoperation/callsc/callsc.go

```diff
@@ -74,6 +75,10 @@ func (c *CallSC) Message() []byte {
 	nbBytes = binary.PutUvarint(buf, c.coins)
 	msg = append(msg, buf[:nbBytes]...)
 
+	// target address length
+	nbBytes = binary.PutUvarint(buf, uint64(len(c.address)))
+	msg = append(msg, buf[:nbBytes]...)
+
```

Comment:
why add this? does the node can handle it?

Reply:
i don't know if the node ever decode this message, but there is no way to decode the address without adding the target address length to the message , and you can check the unit tests that the serialization and deserialization is working with this added  target address length .

Reply:
The api e2e tests are failing with "Failed target_addr deserialization".

Reply:
> there is no way to decode the address without adding the target address length to the message 

there is one in rust, on massa core side, so we must find this same way in golang